dev_guide/builds.adoc

```diff
@@ -1359,7 +1382,53 @@ Configuration change triggers currently only work when creating a new
 able to launch a build whenever a `*BuildConfig*` is updated.
 ====
 
-[[using-docker-credentials-for-pushing-and-pulling-images]]
+
+[build-post-actions]
+== Build Post Actions
+Trigger downstream builds after build completion.
+Triggering any of the above actions in a different namespace is possible as long
+as the user is allowed to start builds in the said namespace.
+
+[[on-success]]
+
+=== On Success Post Action
+This will run once the build succeeded triggering the build of one or
+more specified images.
+
+----
+"onSuccess": [
+  {
+    "type": "StartBuild",
+    "build": {
+      "kind": "BuildConfig",
+       "name": "downstream-buildconfig-1", <1>
+       "namespace": "downstream-bc-namespace" <2>
+    }
+  }
+]
+----
+<1> Specify the buildconfig that should be triggered when the current image is
+successful.
+<2> Here we specify the namespace where this build should be triggered.
```

Comment:
As far I can understand, namespace is optional, right? May we should write it explicitly?